src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int DATA_DATE_POSITION = 0;
+    private static final int DATA_HOUR_POSITION = 2;
+    private static final int DATA_SAlARY_POSITION = 3;
+    private final StringBuilder stringBuilder = new StringBuilder();
```

Comment:
String builder - is not a service class. It is just an improved version of the string. Let's create this object inside the method

Reply:
Moved 2 method.
+fixed in new commit.